modules/setting/setting.go

```diff
@@ -424,6 +425,7 @@ func NewContext() {
 		LandingPageURL = LANDING_PAGE_HOME
 	}
 
+	SSH.ServerCiphers = sec.Key("SSH_SERVER_CIPHERS").Strings(",")
```

Comment:
To keep in order, please move this line of code below the line `SSH.RootPath = path.Join(homeDir, ".ssh")`.